### DISCUSSION TITLE
Toggling the terminal with the same keyboard shortcut? ctrl + ~ How do I achieve this?

### DISCUSSION THREAD

**Discussion by fayazara** [zed-industries/zed#9042] [Feature Requests]
I want to toggle the terminal with the same shortcut as my vscode setup - `ctrl + ~`

How do I set this? 

Right now `⌃ + ~` opens the terminal and `⌘ + ~` closes it. I want to just toggle open/hide with the same shortcut.

**Comment by Moshyfawn** [comment]
There is a `workspace::ToggleBottomDock` action which is bound to `cmd-j` by default. You can [customise](https://zed.dev/docs/key-bindings#adding-a-custom-key-binding
) it to any combination of keys you like by adding it to your `keymap.json` _(`cmd-shift-p` to open the command palette -> `zed: open keymap`)_.

```json
[
  {
    "context": "Workspace",
    "bindings": {
      "ctrl-`": "workspace::ToggleBottomDock"
    }
  }
]
```

It was also discussed in #8342.

**Comment by fpigeonjr** [comment]
just what I needed

**Comment by g14a** [comment]
Saved my fuckin life amazing!

**Comment by JosephTLyons** [comment]
Thanks for reaching out. We want to better support our community, and the best way for us to do that is through our official Discord. The team is active there and happy to help.

We’re closing the “Help and General Q&A” category on GitHub Discussions and directing all support questions to Discord going forward.

https://zed.dev/community-links
